### DISCUSSION TITLE
Ideas for a hybrid modality

### DISCUSSION THREAD

**Discussion by jducoeur** [nescalas/nescala-2023-planning#27] [Ideas]
Collecting ideas for how to craft a "hybrid" conference, with a significant online component.  Add ideas as comments.

**Comment by jducoeur** [comment]
From Geoff Knauth in NE Scala Slack: "A way to make sure the online attendees get something special it to make sure in Q&A sessions, some questions are always pulled from remote visitors."

**Comment by jducoeur** [comment]
Different price tiers.  Online memberships can and probably should be significantly cheaper than in-person.

**Comment by caente** [comment]
definitely a common room for all online *and* in-person folk to hangout together, and perhaps _all_ questions should be read from there, i.e. no more raised hands...

**Comment by jducoeur** [comment]
Check out sli.do for Q&A.

**Comment by jducoeur** [comment]
Suggestion from epishkin:

one of ways to make the experience same to onsite and online attendees could be to use zoom for all speakers. Onsite and remote. Slides and face will be better captured. Not sure what could be done to the sound for onsite speakers...

do we need a camera man when every speaker uses zoom? not sure

but good internet at the venue will be required :100:

**Comment by jducoeur** [comment]
One of the things that worked really well for Scala Love: official breakout rooms per-track, for Q&A and general discussion after each panel.  Basically, a formalized representation of the way that folks crowd around the speaker after many talks.  A bit labor-intensive, since you want a moderator per room, but seemed to work well.
